roles-in-tech/dev-ops.md

```diff
@@ -17,75 +13,70 @@ Here are links to lessons that should be completed before this lesson.
+- [Docker](https://www.docker.com/)is an instrument that helps with packaging code into self-contained units, i.e. containers.
+- [Git](https://git-scm.com/)is a Version Control System with a repository for source code management that enables working online and offline.
+- [Jenkins](https://jenkins.io/) is a tool to use either as a server for continuous integration or a continuous delivery hub that comes with lots of additional plugins to tweak continuous delivery workflow.
+- [Nagios](https://www.nagios.org/) is an infrastructure monitoring tool that presents analytics in visual reports
+- [Selenium](https://www.seleniumhq.org/) is an automated browser that allows QA teams to write scripts and test web products. It’s compatible with eight popular programming languages.
 
-Build on the first information. Have apprentices guess things, do an activity, etc.
+- *For better understanding of **DevOps** refer to this slide [Roles in Tech:DevOps](https://docs.google.com/presentation/d/1GaoHCdvQ2UplDjapp2wG-VRsxlqtzFqIpOtIqvKNYOo/edit?usp=sharing)*
 
-Make sure to mention these things:
+### Lesson
 
-- Things
-	- This is a sub-thing
-- More things
-- Even more things
-- Even more things
+**_DevOps_** stands for development and operations. It’s a practice that aims at merging development, quality assurance, and operations (deployment and integration) into a single, continuous set of processes. This methodology is a natural extension for Agile and continuous delivery approaches.
 
 
 ### Common Mistakes / Misconceptions
 
-List things that apprentices might not realize, might assume at first, or should avoid.
+While many industry leaders have successfully deployed DevOps to achieve faster speed-to-market with higher quality software and a first-class customer experience, there continues to be some fundamental misconceptions about what DevOps really means.
 
-- Example
-- Example
+- **DevOps is a set of automated tools.**
+- **DevOps procedures change from project to project.**: 
 
 
 ### Guided Practice
 
-Have the apprentices work with you as you do something step-by-step.  This can also be fulfilled by a detailed tutorial intended for beginners.
-
+Devops consists of a lot of tools. From learning *Git VCS* to deploying code in *Docker*, one need to learn everything to become a good DevOps Engineer.
+[Edureka](https://www.edureka.co/devops)
 
 ### Independent Practice
 
-Class does this thing themselves with specific additional items. This could be alone, with a partner, or small group; but the idea is that it's less guided, more independent.
+[DevOps best practices](https://devops.com/design-devops-best-practices/)
 
 ### Challenge
 
-Apprentices can try to do this other thing. Ideally, they will be challenged to connect what they've learned to some previous knowledge or additional research.
+[AWS Certified DevOps Engineer - Professional](https://aws.amazon.com/certification/certified-devops-engineer-professional/)
```

Comment:
Same as my last comment for independent practice

Reply:
To be a certified DevOps Engineer , one should refer to the above.